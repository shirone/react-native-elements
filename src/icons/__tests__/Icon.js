import React from 'react';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';
import Icon from '../Icon';

describe('Icon component', () => {
  it('should render without issues', () => {
    const component = shallow(<Icon name="wifi"/>);

    expect(component.length).toBe(1);
    expect(toJson(component)).toMatchSnapshot();
  });
});
