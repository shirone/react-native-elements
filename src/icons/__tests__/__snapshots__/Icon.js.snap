// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon component should render without issues 1`] = `
<View
  style={
    Array [
      false,
      false,
      false,
      Object {
        "alignItems": "center",
        "backgroundColor": "transparent",
        "justifyContent": "center",
      },
      undefined,
    ]
  }
  underlayColor="white"
>
  <Icon
    allowFontScaling={false}
    color="black"
    name="wifi"
    size={24}
    style={
      Array [
        Object {
          "backgroundColor": "transparent",
        },
        undefined,
      ]
    }
  />
</View>
`;
